tjpcov/covariance_builder.py

```diff
@@ -131,26 +153,28 @@ def _build_matrix_from_blocks(self, blocks, tracers_cov):
         s = self.io.get_sacc_file()
         ndim = s.mean.size
 
-        cov_full = -1 * np.ones((ndim, ndim))
+        cov_full = np.zeros((ndim, ndim))
```

Comment:
I'd rather set the zeros after L176 only for the lines that are not selected by the indices. The reason why I put here -1 is that, that way it is easy to see if something went wrong, with 0's you will not see it.